packages/protobuf/scripts/bootstrap-inject.mjs

```diff
@@ -302,15 +309,41 @@ function editionNumbersBetween(
 }
 
 /**
- * @param {FeatureSetDefaults} featureSetDefaults
+ * @param {import("@bufbuild/protobuf/reflect").ReflectMessage} featureSetDefaults - google.protobuf.FeatureSetDefaults
```

Comment:
Because we took some shortcuts in this script, it didn't handle the addition of the feature `enforce_naming_style`. Updated the script to use reflection.